src/Lib.Net.Http.WebPush/PushServiceClientException.cs

```diff
@@ -13,15 +14,36 @@ public class PushServiceClientException : Exception
         /// </summary>
         public HttpStatusCode StatusCode { get; }
 
+        /// <summary>
+        /// Gets or sets the HTTP Response Headers of the push service response.
+        /// </summary>
+        public HttpResponseHeaders Headers { get; set; }
```

Comment:
I would prefer for `Headers`, `Body`, and `PushSubscription` to be readonly properties.

Reply:
Would you also like the existing StatusCode changed to r/o?

Reply:
It is, it has only a getter :).

Reply:
lol, guess I should have looked before I typed :)